include/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics/SatelliteDynamics.hpp

```diff
@@ -0,0 +1,216 @@
+        ///
+        /// @param              [in] anOutputStream An output stream
+        /// @param              [in] (optional) displayDecorators If true, display decorators
+
+        virtual void            print                                       (           std::ostream&               anOutputStream,
+                                                                                        bool                        displayDecorator                            =   true ) const override ;
+
+        /// @brief              Get satellite dynamics's 3 DOF State
+        ///
+        /// @code
+        ///                     State state = satelliteDynamics.getState() ;
+        /// @endcode
+        ///
+        /// @return             State
+
+        State                   getState                                    ( ) const ;
+
+        /// @brief              Set satellite dynamics's 3 DOF State
+        ///
+        /// @code
+        ///                     State state = { ... } ;
+        ///                     satelliteDynamics.setState(state) ;
+        /// @endcode
+        /// @param              [in] aState A 3DOF state
+
+        void                    setState                                    (   const   State&                      aState                                      ) ;
+
+        /// @brief              Obtain dynamical equations function wrapper
+        ///
+        /// @code
+        ///                     Dynamics::DynamicalEquationWrapper dyneq = satelliteDynamics.getDynamicalEquations() ;
+        /// @endcode
+        /// @return             std::function<void(const std::vector<double>&, std::vector<double>&, const double)>
+
+        virtual Dynamics::DynamicalEquationWrapper  getDynamicalEquations   ( )  override ;
+
+    private:
+
+        Environment             environment_ ;
+        Shared<const Frame>     gcrfSPtr_ ;
+        SatelliteSystem         satelliteSystem_ ;
+        State                   state_ ;
+
+        // Only currently used force model that incorporates only Earth's gravity
+        void                    DynamicalEquations                          (   const   Dynamics::StateVector&      x,
+                                                                                        Dynamics::StateVector&      dxdt,
+                                                                                const   double                      t                                           ) ;
+
+        // // Atmospheric perturbations only
+        // void                    Exponential_Dynamics                        (   const   SatelliteDynamics::StateVector&     x,
+        //                                                                                 SatelliteDynamics::StateVector&     dxdt,
+        //                                                                         const   double                                                                  ) const ;
```

Comment:
TBR

Reply:
Does it hurt to keep it for now? Just cause I don't want to lose that code since it can easily be transferred over to a physics atmosphere. There are examples of uncommented code in orbit as well.